nuget/nuget.vcxproj

```diff
@@ -22,7 +22,12 @@
     <VCProjectVersion>15.0</VCProjectVersion>
     <ProjectGuid>{5C91D4CC-745F-4369-85D2-3EC5FDF21FE8}</ProjectGuid>
     <RootNamespace>nuget</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.16299.0</WindowsTargetPlatformVersion>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(WindowsTargetPlatformVersion)'==''">
+    <!-- Latest Target Version property -->
+    <LatestTargetPlatformVersion>$([Microsoft.Build.Utilities.ToolLocationHelper]::GetLatestSDKTargetPlatformVersion('Windows', '10.0'))</LatestTargetPlatformVersion>
```

Comment:
ah cool, this is fancy. 🚀

Reply:
It is so annoying that the build breaks simply because the SDK installed is newer so I'll use this in my C/C++ projects [context](https://docs.vigem.org/#!automatically-select-latest-windows-10-sdk.md).